README.md

```diff
@@ -261,6 +262,7 @@ Feel free to add information about your own db-tools or your favorite third-part
 
 ## Schema samples
 - [Oracle Database Sample Schemas](https://github.com/oracle/db-sample-schemas) - Oracle Database Sample Schemas.
+- [Weaviate examples](https://github.com/semi-technologies/weaviate-examples) - A list of schemas and datasets for the [Weaviate](https://github.com/semi-technologies/weaviate) vector search engine.
```

Comment:
Note to [contributing.md](https://github.com/mgramin/awesome-db-tools/blob/master/contributing.mdl) please - `Submit only one link at the time`